Sources/MessagingInApp/Gist/Managers/MessageQueueManager.swift

```diff
@@ -59,7 +80,11 @@ class MessageQueueManager {
         localMessageStore.removeValue(forKey: queueId)
     }
 
-    private func addMessageToLocalStore(message: Message) {
+    func getInlineMessages(forElementId elementId: String) -> [Message] {
+        localMessageStore.filter { $0.value.elementId == elementId }.map(\.value)
+    }
```

Comment:
Returns all inline messages with a matching element id.

Reply:
what is the use case where we can have multiple messages matching the same element id?

Reply:
[Gist's backend service can return multiple in-app messages with the same element id. You have to use the priority property on the message](https://customerio.slack.com/archives/C036FTE6UJ0/p1716306131792019?thread_ts=1716304555.465419&cid=C036FTE6UJ0). 

I will modify this function in this PR to sort the array by priority before returning.

Reply:
[List is sorted by priority before being returned](https://github.com/customerio/customerio-ios/pull/716/files#diff-ec01291f32e74af14510929ca3d297f876bb1882bd4fdd5008cadca4dff331bcR73).